src/api/ApiService.js

```diff
@@ -22,7 +22,7 @@ class ApiService {
      * @returns {import('firebase/firestore').DocumentReference<import('./types').User>}
      */
     user(userID) {
-        return doc(this.users, userID);
+        return userID ? doc(this.users, userID) : null;
```

Comment:
Shouldn't like the firebase just return undefined/null or even don't run a request if we don't have an Id? If it's not the firebase is really stupid lmao

Reply:
No, firebase is retarded it throws an error, maybe it is not actually used and is only related to some kind of work in progress bug fix or sth.